config/application.rb

```diff
@@ -47,5 +47,9 @@ class Application < Rails::Application
         password: ENV['SMTP_PASSWORD'],
       }
     end
+
```

Comment:
Just a heads up @tdooner. My tests starting flaking like hell when I also didn't also add
```
routes.default_url_options = {host: ENV['APP_DOMAIN']}
```

No clue why but just want to save you the headache if you see flakiness around this config.